prov/efa/test/README.md

```diff
@@ -0,0 +1,61 @@
+# EFA unit tests
+
+## How to run
+
+To run efa unit tests, you will need to have cmocka installed.
+See: https://cmocka.org/
+
+You will need to configure libfabric with ```--enable-efa-unit-test```.
```

Comment:
A high level question: Does this unit test require efa device? If I configure libfabric with such flag and `make check` on a node that does not have efa device, will the test fail?

Reply:
Tests must have changed... now they do.

Reply:
Currently, an EFA device is required.